Assets/Tests/InputSystem/APIVerificationTests.cs

```diff
@@ -575,6 +576,187 @@ public void API_MonoBehavioursHaveHelpUrls()
         Assert.That(brokenHelpUrls, Is.Empty);
     }
 
+    private const string kAPIDirectory = "Tools/API";
+
+    ////FIXME: The .api-based checks are temporary and don't account for platform-specific APIs. Nuke these tests as soon
+    ////       as we can switch back to API validation performed by the Package Validation Suite (as soon as Adriano's fix
+    ////       for the access modifier false positive has landed).
```

Comment:
Talked this over with Adriano. He'll go and fix the false positive and PR the change into Package Validation Suite. When that change is out, we can probably switch back to using API Validation from PVS.

However, with the exclusion mechanism that PVS currently has, we have no ability to exclude individual breaking changes. So could be, that we'll run into another problem that we can't exclude.

I opted to just run with this test for now, as imperfect as it is. At least it gives us back control and so far, the detections it comes up with all seem sensible.

Reply:
I realize now that we'll probably have to live with this API test instead of relying on API Validation even after @adrianoc-unity3d's fix has landed. Problem is, we will need to slap the preview label back on our package instead of going directly to 1.1. Adding the label back will bypass API Validation checks from what I know. So seems that even then, we'll still need our own thing :(